snorql-framework/pom.xml

```diff
@@ -4,13 +4,13 @@
     <parent>
         <artifactId>snorql</artifactId>
         <groupId>com.udaan.snorql</groupId>
-        <version>0.6-SNAPSHOT</version>
+        <version>0.6.1-SNAPSHOT</version>
```

Comment:
I think this should be a minor version bump.
Please go through this article - https://medium.com/@amit.desai03/semantic-versioning-bda7a1c5b43f